flocker/node/backends.py

```diff
@@ -90,6 +90,23 @@ class BackendDescription(PClass):
     :ivar deployer_type: A constant from ``DeployerType`` indicating which kind
         of ``IDeployer`` the API object returned by ``api_factory`` is usable
         with.
+
+    The following fields are experimental, and subject to change
```

Comment:
:question: Maybe put these fields behind an interface. `IBackendProvisioner` or something. Just so there are more places you could put the warning.